modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/constants/WorkflowInstancePortletConstants.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.workflow.web.internal.constants;
+
+/**
+ * @author Janis Zhang
+ */
+public class WorkflowInstancePortletConstants {
```

Comment:
@JanisZhang Could you rename this class to `WorkflowInstancePortletWebKeys`? A constant that is used as request attribute key should be put in `WebKeys` class; a lot of other modules also have both `*Constants` and `*WebKeys` classes at the same time.